gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageOptions.java

```diff
@@ -198,6 +199,8 @@ public static Builder builder() {
 
   public abstract MetricsSink getMetricsSink();
 
+  public abstract Boolean getTraceLogEnable();
```

Comment:
Getters for boolean fields should have `is` prefix (see other configs)

Reply:
Done